src/Microsoft.Identity.Web.UI/Areas/MicrosoftIdentity/Controllers/AccountController.cs

```diff
@@ -39,11 +39,15 @@ public AccountController(IOptionsMonitor<MicrosoftIdentityOptions> microsoftIden
         /// </summary>
         /// <param name="scheme">Authentication scheme.</param>
         /// <param name="redirectUri">Redirect URI.</param>
+        /// <param name="loginHint">Login hint (user's email address).</param>
+        /// <param name = "domainHint" > Domain hint.</param>

```

Comment:
```suggestion
        /// <param name="domainHint">Domain hint.</param>

```
nit: consistent spacing